### DISCUSSION TITLE
Filtering table based on form selection with other schema

### DISCUSSION THREAD

**Discussion by henricbladh** [Budibase/budibase#9263] [Help] [answered]
Hi, I have a form and then two tables on one page. I would like to filter the tables based on the user selection in a relationship selector in the form. Here is a picture of the page.
![Screenshot 2023-01-06 at 18-11-37 Budibase](https://user-images.githubusercontent.com/17470021/211052270-87f41b2e-ebd3-407a-84ea-62cf5a36822b.png)

If the user selects e.g. 2 in the Equipment No the first dropdown the first table should be empty and the second should display the row for Equipment No 2.

I have three BB tables which all have the Equipment No field related to a Number field in a fourth table.

Is this possible to do? If so, how?
TIA

**Comment by melohagan** [comment, marked as answer]
Hey @henricbladh 

I did create a guide on how to do this here: https://docs.budibase.com/docs/filter-by-relationships

Unfortunately there is a bit of a bug with it, but the workaround is given in the issue: https://github.com/Budibase/budibase/issues/9015

**Reply by melohagan** [reply]
Actually if you're using the Internal DB then following that guide should work fine. The decoding is only needed for external SQL databases.
